bundles/org.openhab.binding.mybmw/src/test/java/org/openhab/binding/mybmw/internal/dto/charge/ChargingStatisticsTest.java

```diff
@@ -118,7 +119,7 @@ private boolean testVehicle(String statusContent, int callbacksExpected,
             Method updateChargeStatisticsMethod = VehicleHandler.class.getDeclaredMethod("updateChargingStatistics",
                     ChargingStatisticsContainer.class, String.class);
             updateChargeStatisticsMethod.setAccessible(true);
-            updateChargeStatisticsMethod.invoke(vehicleHandler,
+            updateChargeStatisticsMethod.invoke(Objects.requireNonNull(vehicleHandler),
```

Comment:
Due to limitations the null analysis has no idea what happens in method implementations being called. It only considers annotations on other methods and null checks in the current method. It would be very time consuming for the analysis to check each every method calling another method dozens of levels deep including JVM, framework and library code. 🙃